test/functional/apps/saved_objects_management/edit_saved_object.ts

```diff
@@ -70,113 +41,34 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       );
     });
 
-    it('allows to update the saved object when submitting', async () => {
+    it('allows to view the saved object', async () => {
       await PageObjects.settings.navigateTo();
       await PageObjects.settings.clickKibanaSavedObjects();
-
-      let objects = await PageObjects.savedObjects.getRowTitles();
+      const objects = await PageObjects.savedObjects.getRowTitles();
       expect(objects.includes('A Dashboard')).to.be(true);
-
-      await PageObjects.common.navigateToUrl(
-        'management',
-        'kibana/objects/savedDashboards/i-exist',
-        {
-          shouldUseHashForSubUrl: false,
-        }
-      );
-
-      await testSubjects.existOrFail('savedObjectEditSave');
-
-      expect(await getFieldValue('title')).to.eql('A Dashboard');
-
-      await setFieldValue('title', 'Edited Dashboard');
-      await setFieldValue('description', 'Some description');
-
-      await focusAndClickButton('savedObjectEditSave');
-
-      objects = await PageObjects.savedObjects.getRowTitles();
-      expect(objects.includes('A Dashboard')).to.be(false);
-      expect(objects.includes('Edited Dashboard')).to.be(true);
-
-      await PageObjects.common.navigateToUrl(
-        'management',
-        'kibana/objects/savedDashboards/i-exist',
-        {
-          shouldUseHashForSubUrl: false,
-        }
-      );
-
-      expect(await getFieldValue('title')).to.eql('Edited Dashboard');
-      expect(await getFieldValue('description')).to.eql('Some description');
+      await PageObjects.common.navigateToUrl('management', 'kibana/objects/dashboard/i-exist', {
+        shouldUseHashForSubUrl: false,
+      });
+      const inspectContainer = await find.byClassName('kibanaCodeEditor');
+      const visibleContainerText = await inspectContainer.getVisibleText();
+      // ensure that something renders visibly
+      expect(visibleContainerText.includes('A Dashboard'));
```

Comment:
Should we assert that the content of the `kibanaCodeEditor` is a JSON with the expected keys?

Reply:
We're adding line numbers so trying to parse the `visibleContainerText` throws an `SyntaxError: Unexpected number in JSON at position 2`. The underlying monaco-editor displays each line in a `div` that wraps nested `spans`. e.g.
![Screen Shot 2021-09-16 at 16 23 24](https://user-images.githubusercontent.com/11909450/133698882-45ea8e9e-2540-4466-a36d-665361c40a12.png)
Would it be enough to check that the keys we expect to be there are?
i.e something along the lines of:
```
const textIncludesAll = (text: string, items: string[]) => {
    const bools = items.map((item) => !!text.includes(item));
    return bools.every((currBool) => currBool === true);
  };
expect(textIncludesAll(visibleContainerText, keysExpected).to.be(true);
```